### PR TITLE
FIX(PN-13267) Fix void page

### PR DESCRIPTION
**Pull request by leleOFA**
fix: delete margin in bottom of page in AOT

### CODE REVIEW

File: src/main/resources/documents_composition_templates/NotificationViewedLegalFact_fr.html [hunk 1]
```diff
@@ -127,7 +127,7 @@ <h3 class="document-type">
...
+    <p class="caption my-0">L’attestazione riporta la data in cui il destinatario (o il suo delegato) per la prima volta ha avuto accesso tramite la piattaforma al documento oggetto di notificazione. Tale data non necessariamente corrisponde alla data di perfezionamento della notifica, che può anche essere antecedente per decorrenza termini. Ai fini di individuare la giusta data di perfezionamento della notifica, si invita il destinatario (o il suo delegato) a verificare <strong><a class="faq-link-perfezionamento" href="https://notifichedigitali.pagopa.it/faq#perfezionamento-come-posso-sapere">in questa FAQ</a></strong> come deve essere calcolata la data di perfezionamento.</p>
```
Comment (SarahDonvito) [inline src/main/resources/documents_composition_templates/NotificationViewedLegalFact_fr.html:130]: it's better to remove only margin bottom and leave mt-8

**Review by SarahDonvito — CHANGES_REQUESTED** [review]

**Review by SarahDonvito — CHANGES_REQUESTED** [review]
We should check other ao3 templates too

File: src/main/resources/documents_composition_templates/NotificationViewedLegalFact_fr.html (hunk 1 above)
Comment (leleOFA) [reply src/main/resources/documents_composition_templates/NotificationViewedLegalFact_fr.html]: done

File: src/main/resources/documents_composition_templates/NotificationViewedLegalFact.html [hunk 2]
```diff
@@ -108,18 +108,18 @@ <h3 class="document-type">
...
+    <p class="caption mt-8">L’attestazione riporta la data in cui il destinatario (o il suo delegato) per la prima volta ha avuto accesso tramite la piattaforma al documento oggetto di notificazione. Tale data non necessariamente corrisponde alla data di perfezionamento della notifica, che può anche essere antecedente per decorrenza termini. Ai fini di individuare la giusta data di perfezionamento della notifica, si invita il destinatario (o il suo delegato) a verificare <strong><a class="faq-link-perfezionamento" href="https://notifichedigitali.pagopa.it/faq#perfezionamento-come-posso-sapere">in questa FAQ</a></strong> come deve essere calcolata la data di perfezionamento.</p>
```
Comment (SarahDonvito) [inline src/main/resources/documents_composition_templates/NotificationViewedLegalFact.html:122]: we should use `mb-0` and not `mt-8`

File: src/main/resources/documents_composition_templates/PecDeliveryWorkflowLegalFact_fr.html [hunk 3]
```diff
@@ -157,10 +157,10 @@ <h3 class="document-type">
     <p class="right"><span data-hide-on-it="true">Signé numériquement par PagoPA S.p.A. &bull; </span>Firmato digitalmente da PagoPA S.p.A.</p>
 </div>
 <div class="row margin-top-small">
-    <div class="left-col col-4-10">
+    <div class="left-col col-3-10">
         <p>&nbsp;</p>
     </div>
-    <div class="left-col col-6-10">
+    <div class="left-col col-77-10">
```
Comment (SarahDonvito) [inline src/main/resources/documents_composition_templates/PecDeliveryWorkflowLegalFact_fr.html:163]: pay attention to typo `col-7-10`

**Review by SarahDonvito — CHANGES_REQUESTED** [review]

File: src/main/resources/documents_composition_templates/PecDeliveryWorkflowLegalFact_fr.html (hunk 3 above)
Comment (leleOFA) [reply src/main/resources/documents_composition_templates/PecDeliveryWorkflowLegalFact_fr.html:163]: dopne

File: src/main/resources/documents_composition_templates/NotificationViewedLegalFact.html (hunk 2 above)
Comment (leleOFA) [reply src/main/resources/documents_composition_templates/NotificationViewedLegalFact.html:122]: done

File: src/main/resources/documents_composition_templates/NotificationCancelledLegalFact.html [hunk 4]
```diff
@@ -86,10 +86,10 @@ <h3 class="document-type">Dichiarazione annullamento notifica sulla piattaforma
 </div>
 
 <div class="row">
-    <div class="left-col col-4-10">
+    <div class="left-col col-7-10">
```
[the remark this answers is not in the document]
Comment (leleOFA) [reply src/main/resources/documents_composition_templates/NotificationCancelledLegalFact.html:89]: done

File: src/main/resources/documents_composition_templates/NotificationViewedLegalFact_fr.html (hunk 1 above)
Comment (SarahDonvito) [reply src/main/resources/documents_composition_templates/NotificationViewedLegalFact_fr.html]: use `mb-0` because `my-0` doesn't exist in css

**Review by SarahDonvito — APPROVED** [review]

**Pull request merged**